validator_client/src/main.rs

```diff
@@ -43,6 +43,15 @@ fn main() {
                 .help("Address to connect to BeaconNode.")
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("spec")
+                .long("spec")
+                .value_name("spec")
+                .short("s")
+                .help("Configuration of Beacon Chain")
+                .takes_value(true)
+                .possible_values(&["foundation", "few_validators"]),
```

Comment:
I think it would be cool to specify [`default_value`](https://docs.rs/clap/2.32.0/clap/struct.Arg.html#method.default_value) here, so the CLI will tell you which is the default.

Example:
`--log-level <LOG_LEVEL>    Logging level. [default: debug]  [possible values: error, warn, info, debug, trace]`

Reply:
ok :+1: